dhall-docs/dhall-docs.cabal

```diff
@@ -22,26 +22,39 @@ Description:
 Category: Compiler
 Extra-Source-Files:
     CHANGELOG.md
+Data-Files:
+    src/Dhall/data/*.css,
+    src/Dhall/data/*.ttf
```

Comment:
Perhaps there could be an option to embed these data files in the executable (for ease of distributing standalone executables, such as on our Releases page).  `pandoc` and `criterion` both have an an `-fembed-data-files` configuration flag to do something similar, so you can study either one of them to see how it is done